systers_portal/meetup/views.py

```diff
@@ -19,3 +20,18 @@ class MeetupLocationList(ListView):
     template_name = "meetup/list_location.html"
     model = MeetupLocation
     paginate_by = 20
+
+
+class MeetupView(MeetupLocationMixin, DetailView):
```

Comment:
We probably need to do this:

`meetup = get_object_or_404(slug=self.kwargs['meetup_slug'], meetup_location=self.object)` and not filter (I didn't pay much attention at first). And then add it to the context. This way we don't raise any exceptions. And in the tests you just check the 404 status code.

Does that make sense?